runelite-client/src/main/java/net/runelite/client/plugins/stealingartefacts/StealingArtefactsOverlay.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2019, honeyhoney <https://github.com/honeyhoney>
+ * Copyright (c) 2020, ThePharros <https://github.com/ThePharros>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.stealingartefacts;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Graphics2D;
+import javax.inject.Inject;
+import net.runelite.client.ui.overlay.Overlay;
+import net.runelite.client.ui.overlay.OverlayPosition;
+import net.runelite.client.ui.overlay.OverlayPriority;
+import net.runelite.client.ui.overlay.components.PanelComponent;
+import net.runelite.client.ui.overlay.components.TitleComponent;
+
+class StealingArtefactsOverlay extends Overlay
+{
+	private final StealingArtefactsPlugin plugin;
+	private final PanelComponent panelComponent = new PanelComponent();
+
+	@Inject
+	private StealingArtefactsOverlay(StealingArtefactsPlugin plugin)
+	{
+		super(plugin);
+		setPosition(OverlayPosition.TOP_LEFT);
+		setPriority(OverlayPriority.LOW);
+		this.plugin = plugin;
+	}
+
+	@Override
+	public Dimension render(Graphics2D graphics)
+	{
+
+		StealingArtefactState stealingArtefactState = plugin.getStealingArtefactState();
+
+		if (!plugin.isInPortPiscariliusRegion() || stealingArtefactState == null)
```

Comment:
When using the north escape (for the west and southwest houses), you wander slightly outside the defined port piscarilius regions. It would be nice to continue to show the overlay in that case:
```suggestion
		if (stealingArtefactState == null || (stealingArtefactState != StealingArtefactState.DELIVER_ARTEFACT && !plugin.isInPortPiscariliusRegion()))
```